README.md

```diff
@@ -64,17 +53,10 @@ install it with Homebrew:
 brew install postgresql
 ```
 
-Install JS dependencies and start Pursuance's auto-reloading dev server:
-
-```
-cd $(go env GOPATH)/src/github.com/PursuanceProject/pursuance
-npm install
-npm run build
-npm run start
-```
+Next, you'll need at least three terminals.
```

Comment:
`you'll need at least three terminals` -> `you'll need three terminals`

Reply:
:heavy_check_mark: